src/store/index.js

```diff
@@ -1,10 +1,12 @@
 import Vue from 'vue'
 import Vuex from 'vuex'
+import VueHtml2pdf from 'vue-html2pdf'
 import { Configuration } from '@/configuration'
 import metrics from '@/store/metrics'
 import AsyncComputed from 'vue-async-computed'
 
 Vue.use(Vuex)
+Vue.use(VueHtml2pdf)
 Vue.use(AsyncComputed)
```

Comment:
I would prefer to have this in app.js or in a new file imported from app.js. Unlike Vuex, these plugins are not dedicated to the store.